site/_includes/docs/_docs-header.liquid

```diff
@@ -0,0 +1,23 @@
+<div data-animation="default" data-collapse="medium" data-duration="400" data-easing="ease" data-easing2="ease" role="banner" class="main-nav-bar w-nav">
+    <div class="main-nav-container w-container"><img src="/assets/img/docs/landing/fusionauth-docs-logo.svg" loading="lazy" alt="fusionauth logo" class="fusionauth-gray-logo">
```

Comment:
This logo should take you back to `/docs/` otherwise there's no way to do so.

Reply:
Agreed.